### PR TITLE
Issue Explorer: Moving expiry date to flyout.

### LINKED ISSUE #2962: Issue Explorer: Move Expiry Date to Flyout
### Current Behavior

Issue Explorer looks a little busy with all the info below. It would be great to make it less busy and move less important info to the flyout
screenshot 2018-11-28 01 22 07

### Expected Behavior
1. Remove 'Expires in X days'
screenshot 2018-11-28 01 23 43

2. Move expiry date to flyout, with a clock icon
flyoutexpiry

Sketch file:
https://drive.google.com/open?id=1ssPVGV4_WVVrfPsEaR7mhm4lzYcc631d

### PR DESCRIPTION
**Pull request by Anish-Agnihotri**
##### Description

Simple PR to move the expiry date from the bounty-row in explorer to the flyout. 

##### Checklist

- [X] temporary clock icon removed
- [X] linter status: 100% pass
- [X] changes don't break existing behavior
- [X] commit message follows [commit guidelines](https://docs.gitcoin.co/mk_contributors/#step-4-commit)

##### Affected core subsystem(s)

UI. 

##### Testing

Tested across devices and resolutions as usual.  

##### Refers/Fixes

Fixes #2962.

### CODE REVIEW

File: app/dashboard/templates/shared/bounty-popover.html [hunk 1]
```diff
@@ -44,6 +44,9 @@
         [[/for]]
       [[/if]]
     </div>
+    <div class="mt-2">
+      [[if expiredIcon === "true"]]<img src="{% static "v2/images/time-expired.svg" %}" class="expired-icon" />[[/if]][[:expired]]
```
Comment (pinkiebell) [inline app/dashboard/templates/shared/bounty-popover.html]: You can save one one property by checking just for `expired` 👍

**Comment by Anish-Agnihotri** [comment]
Thank you for reviews @thelostone-mc and @pinkiebell. 

Will push out revised PR to spec tomorrow morning and bump thread for merge :).

Thanks!

Anish

**Comment by octavioamu** [comment]
@willsputra is this how it looks with the fontawesome one
image

**Comment by Anish-Agnihotri** [comment]
@octavioamu @willsputra 

Tried throwing in font-awesome real quickly so you could see it in action live. Here is an image of it: 

font-awesome

I'll update this branch to the font-awesome changes, but I've also finished the changes with the SVG in a local branch.

Let me know which one is best for you, and I'll make another push appropriately (if need be).

File: app/dashboard/templates/shared/bounty-popover.html (hunk 1 above)
Comment (Anish-Agnihotri) [reply app/dashboard/templates/shared/bounty-popover.html]: Thanks :)! That was indeed a better idea. Just refactored and removed the `expiredIcon` property.

**Comment by octavioamu** [comment]
@Anish-Agnihotri lets go with font awesome, in my print I used font-size 15px to be the same size as the svg one.

File: app/assets/v2/css/dashboard.css [hunk 2]
```diff
@@ -269,3 +269,10 @@ body {
 .btn-expand-token:focus {
   color: #007bff;
 }
+
+.expired-icon {
```
Comment (octavioamu) [inline app/assets/v2/css/dashboard.css]: with the changes to font I think only the font-size will be enough maybe margin also.

Comment (Anish-Agnihotri) [reply app/assets/v2/css/dashboard.css]: Sure! Changed the `font-size` to `15px`, and kept the margin (to seperate the clock icon from the following text). Also kept the transform `translateY(5%);` though. Without it, the clock is off center in comparison to the text.

On another hand, I can up the font-size of the text as well and send an image if you'd like.

Comment (thelostone-mc) [reply app/assets/v2/css/dashboard.css]: @Anish-Agnihotri nah this should be alright ^_^

Comment (octavioamu) [reply app/assets/v2/css/dashboard.css]: Just a minor thing but transformations are heavy tasks for the browser, I believe we should use vertical align or other text focus css properties.

Comment (Anish-Agnihotri) [reply app/assets/v2/css/dashboard.css]: Sure, that’s not a problem at all. I’ll jump in and remove the transform and move over to padding or another text focus property. I’ll push another PR in ~2hrs, not at home atm.

Comment (octavioamu) [reply app/assets/v2/css/dashboard.css]: Sorry `vertical-align: text-bottom;` to be exact the same position of the transform

**Pull request merged**